bin/load-spark-env.sh

```diff
@@ -65,6 +65,6 @@ export SPARK_SCALA_VERSION=2.13
 #fi
 
 # Append jline option to enable the Beeline process to run in background.
-if [[ ( ! $(ps -o stat= -p $$) =~ "+" ) && ! ( -p /dev/stdin ) ]]; then
+if [ -e /usr/bin/tty -a "`tty`" != "not a tty" -a ! -p /dev/stdin ]; then
```

Comment:
```suggestion
if [ -e /usr/bin/tty ] && [ "$(tty)" != "not a tty" ] && [ ! -p /dev/stdin ]; then
```

Reply:
ref https://github.com/koalaman/shellcheck/wiki/SC2166 and https://github.com/koalaman/shellcheck/wiki/SC2006